FWCore/MessageLogger/src/JobReport.cc

```diff
@@ -302,18 +302,40 @@ namespace edm {
     }
   }
 
+  static char const* const kJobReportEndElement = "</FrameworkJobReport>\n";
+  static constexpr int kEndElementSize = 22;
```

Comment:
Would it be possible and feasible to use `std::string_view` and take the number of elements from its `size()` (which, I see, should be `constexpr`)?